gammapy/time/lightcurve.py

```diff
@@ -320,6 +321,136 @@ def create_fixed_time_bin(time_step, spectrum_extraction):
             ])
         return intervals
 
+    def create_fixed_significance_bin_lc_table(self, significance, significance_method, energy_range,
+                                               spectrum_extraction, separators=[]):
+        """
+
+        Create time intervals such that each bin of a light curve reach a given significance and store information
+
+        Parameters
+        ----------
+        significance : float
+            Target significance for each light curve point
+        significance_method : {‘lima’, ‘simple’}
```

Comment:
From copy & pasting from the HTML docs you have non-ascii quote characters here.
Put normal single quotes:
```
        significance_method : {'lima', 'simple'}
```